tests/runtests.pl

```diff
@@ -3865,7 +3865,7 @@ sub singletest {
     if($cmdtype eq "perl") {
         # run the command line prepended with "perl"
         $cmdargs ="$cmd";
-        $CMDLINE = "perl ";
```

Comment:
I should probably even be just

    $CMDLINE = $perl;

... since $perl is already set to be a suitable command line to invoke perl with. What do you think?

Reply:
I considered that, but figured they were separated for a reason. But it sounds like that isn't the case. I'll update the pull request accordingly.